### PR TITLE
HTML/CSS & JavaScript capstone project - Conference page

### PR DESCRIPTION
**Pull request by AlselwiAisha**
- Build home page and about page.
- Have versions for 2 different screen sizes: Mobile and Desktop.
- The home page has a link in the menu to the about page.
- The logo in the header links to the home page.
- Added mobile menu.
- Sectors and members sections created dynamically in JavaScript.
- [Live Demo](https://alselwiaisha.github.io/Capstone-project/)

### CODE REVIEW

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @AlselwiAisha   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter error :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [x] Kindly make sure to wrap your top navigation bar into the `nav` tag which will increase the page accessibility. Please remember to edit both files `index.html` and `about.html`.
       
       
   image

- [x] Kindly make sure to align the `footer` contents in the center of the page in order to match the provided Behance design.

   - ### YOUR DESIGN
   
  image

   - ### BEHANCE DESIGN
   
   image

 - [x] Kindly make sure to remove the Partners and Footer section on the mobile version on the Home page. The Behance doesn't show those two sections in the Home page mobile version. Kindly fix that.
  
 - ### YOUR DESIGN
 
   image

 - ### BEHANCE DESIGN
 
   image


- [x] Kindly make sure to use the provided Google font `Lato` and `COCOGOOSE`. In order to use this special font you have to import them from [google fonts](https://fonts.google.com/) and then have access to it in order to use it in your project. Kindly make sure your website's typography matches the provided one on Behance design by using the correct font.


- [x] Kindly make sure to add the red short line which locates under each section title in order to match the provided Behance design as much as possible.

  - ### YOUR DESIGN
  
   image

   image

    image

  - ### BEHANCE DESIGN
  
     image

   image

   image


- [x] Kindly make sure the working title of all speakers is written in the same font style as the Behance design show.

  - ### YOUR DESIGN
  
   image


  - ### BEHANCE DESIGN
  
   image


- [x] Kindly make sure to remove all comments and placeholder text in your README file in order to keep your repository clean and professional.

  
   image


 - [x] Kindly make sure to adjust the border of your button in order to match the provided design as much as possible. 

   - ### YOUR DESIGN
   
   image

   - ### BEHANCE DESIGN
   
    image

- [x] Kindly make sure the paragraphs of the sectors card are aligned on the same straight vertical line in order to match the provided Behance design. 
- [ ] Kindly make sure the typography of the sectors card contents matches the one on the provided Behance design.

  - ### YOUR DESIGN
  
  chrome-capture-2023-4-31

  - ### BEHANCE DESIGN
  
    chrome-capture-2023-4-31 (1)


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -122,17 +121,19 @@ <h2 class="partner-text text-center  pt-5">Success Partners</h2>
...
           <p class="d-none d-md-block footer-desc text-left">A youth- and women-led non-governmental organization (NGO) based in Sana’a, Yemen.
              </p>
        </div> 
     </div>
 </footer>
+    <button class="btn-more d-md-none text-dark p-3"
+    type="button">MORE <i class='bx bx-chevron-down'></i> </button>
 <script src="./JS/script.js"></script>
 </body>
 </html>
```
Comment (usorfaitheloho) [inline index.html]: As suggested by the previous reviewer, please remove all commented lines of code and descriptions from your README file in order to have a clean and legible code 😄

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @AlselwiAisha  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * Awesome design✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (AlselwiAisha) [reply index.html:139]: Sorry, I don't get the issue.

**Review by levy002 — APPROVED** [review]
## Hello @AlselwiAisha 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**